lib/gruf/lightstep/server_interceptor.rb

```diff
@@ -46,7 +46,7 @@ def call(&_block)
           rescue StandardError => e
             span.set_tag('error', true) if error?(e)
             span.set_tag('grpc.error', true)
-            span.set_tag('grpc.error_code', e.code)
+            span.set_tag('grpc.error_code', e.code) if e.respond_to?(:code)
```

Comment:
I think here we should:

* Default to 13, which is `GRPC::Internal`
* Make this a configuration setting, with the name of `default_error_code`

So, something like `e.respond_to?(:code) ? e.code : ::Gruf::Lightstep.default_error_code`